__index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Command-line and other tools for Jena developers
+slug: index
+---
+
+Jena includes various command-line utilities which can help you with
+a variety of tasks in developing Jena-based applications.
+
+### Index of tools
+
+  - [schemagen](schemagen.html)
+  - [using schemagen from maven](schemagen-maven.html)
```

Comment:
I think the project has stopped supporting `schemagen` as of 4.x so not sure we want it in the docs

I know for other retired/defunct modules we change the original docs to explicitly note this e.g. https://jena.apache.org/documentation/hadoop/index.html

Reply:
`schemagen` is still there as a command line tool but you're right the plugin `schemagen-maven` is (from v3.7.0 onwards) retired.

https://github.com/apache/jena/tree/main/jena-maven-tools/README.md

This isn't text that has changed from the original. We need to clearup but separate issue.